src/plugins/intel_cpu/CMakeLists.txt

```diff
@@ -37,6 +37,10 @@ if(TBB_FOUND AND TBB_VERSION VERSION_GREATER_EQUAL 2018 AND TBB_VERSION VERSION_
 endif()
 
 add_subdirectory(thirdparty)
+add_subdirectory(tests/unit_tests)
```

Comment:
Shouldn't both subfolders be wrapped by `ENABLE_TESTS` ?

Reply:
Yes.